.travis.yml

```diff
@@ -89,26 +82,25 @@ script:
 
 jobs:
   include:
-    - osx_image: xcode11.2
+    - osx_image: xcode11.3
       <<: *lang_env
       <<: *homebrew
       <<: *caches
     - osx_image: xcode10.3
     - osx_image: xcode9.4
     - osx_image: xcode7.3
     - stage: deploy
-      osx_image: xcode11.2
+      osx_image: xcode11.3
       <<: *lang_env
       <<: *caches
       script: skip
       before_deploy:
         - make -C src macvim-dmg
       deploy:
         provider: releases
-        api_key:
+        token:
           secure: ukjm+qbuNiTli25Ut2BoVpeBCV+JyVbRUwPqjTKrJxfHz34bpr38eSbryIB8BgKBItgzE876Yoqa3CD0k8mqGClis1+98MtrYFpAkO97juJmHpcZZZB7ausbHGf7Z7VdMT4jBjjVGcBeaNj0mio0hwem0/S4WyJK3M/3Fym995CltCUtJKRfMvRiGkWZqUs8K7EZf53DFR6CXUn38rq/3B88SeK51OZuCkMsiDWLGYCdayH19vJfFrTF8MYMQYDYxz16Q/Kf21PVhwia7HEhOzqnXS8RXS+vLkZw8mzIxowX+w6NT90q7Sj0ENdR7YaS27QPfDdhZEnOgpgqj+za63lpiyIdRcgSBkGxNYrM6B5KhiwC1VocBxCBdCxT5WXlx9rA9+k4CASdsxAW/MtQOK6PRMfZEnAB+ShFvshM2H/iE5Jch+o/SIjCXhdkeASD5qov2x6eXcsEVu8PIxvEUptCpHeqJTN5/26nfKsvOdrsqbwJbDluwISOKfEPhohb8Hn7JqOJNTS2aJr3jfvU+egE1NS0eLqKPXecu7MOOsOq1CQL6WxblphG2JCCmAOuNMYrJx9+w28ekMDRDAbI9r5nWcPLZtBqjFUyuBXXM7UknMar0FZ2fd7YTi/Gki3n56UN0lKaSAKaJB9EXlneDSKp/1ogsETr9/b7jz0s6lI=
         file: src/MacVim/build/Release/MacVim.dmg
-        skip_cleanup: true
```

Comment:
I think this needs to stay. The warning produced by Travis is quite misleading because their validator is validating against v2 of dpl (the deployment tool) and we are using v1 still (v2 is opt-in and experimental only still). Unless we opt-in to v2 (let's not do that for now) by doing `edge:true`, we need to keep the `skip_cleanup: true` flag to prevent Travis from cleaning up therefore making it impossible to release.